pkg/scheduler/plugins/capacity/capacity.go

```diff
@@ -890,22 +891,23 @@ func (cp *capacityPlugin) jobEnqueueable(queue *api.QueueInfo, job *api.JobInfo)
 	// The queue resource quota limit has not reached
 	r := minReq.Clone().Add(attr.allocated).Add(attr.inqueue).Sub(attr.elastic)
 
-	return r.LessEqualWithDimension(attr.realCapability, minReq)
+	return r.LessEqualWithDimensionAndResourcesName(attr.realCapability, minReq)
```

Comment:
Thanks for your work。

I hope we can make this more detailed by including the total quota and the allocated amount.
For example, in native Kubernetes scheduling, when resources are insufficient, the error message looks like:
`InsufficientResources cpu: Req:24000 0/0; memory: Req:257698037760 0/0; nvidia.com/gpu: Req:2 0/0;`